KomaMRICore/test/runtests.jl

```diff
@@ -706,6 +730,9 @@ end
 
 @testitem "BlochDict_CPU_single_thread" tags=[:important, :core] begin
     using Suppressor
+    include("../../KomaMRIIO/src/KomaMRIIO.jl")
+    using .KomaMRIIO
+
```

Comment:
Here

Reply:
Addressed in [a7145d6](https://github.com/cncastillo/KomaMRI.jl/commit/a7145d642a633d7aa5d238007b66511b91241eda)